addons/mail/static/src/components/messaging_menu/messaging_menu.js

```diff
@@ -167,6 +158,14 @@ class MessagingMenu extends Component {
     _onClickToggler(ev) {
         // avoid following dummy href
         ev.preventDefault();
+        if (!this.env.messaging) {
+            /**
+             * Messaging not created, which means essential models like
+             * messaging menu are not ready, so user interactions are omitted
+             * during this (short) period of time.
+             */
+            return;
+        }
```

Comment:
good catch, there are no other suck handlers that could be problematic here or in other root components?

Reply:
It's for handlers that could be called when messaging not ready. Just found this one, and I felt it was nice to be able to open/close messaging menu even when messaging is not yet ready. The tiny short time when `env.messaging` is not yet created ignores the click, which is a bit of a shame but considering models are not ready and it's very unlikely the user has time to click when messaging is not created, I feel it's good enough.